interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSuggester.scala

```diff
@@ -51,20 +51,20 @@ class SpelExpressionSuggester(expressionConfig: ExpressionDefinition[_], typeDef
         case TypingResultWithContext(tc: TypedClass, staticContext) => Future.successful(typeDefinitions.get(tc.klass).map(c => filterClassMethods(c, p.getName, staticContext)).getOrElse(Nil))
         case TypingResultWithContext(to: TypedObjectWithValue, staticContext) => Future.successful(typeDefinitions.get(to.underlying.klass).map(c => filterClassMethods(c, p.getName, staticContext)).getOrElse(Nil))
         case TypingResultWithContext(to: TypedObjectTypingResult, _) =>
-          val suggestionsFromFields = filterMapByName(to.fields, p.getName).toList.map { case (methodName, clazzRef) => ExpressionSuggestion(methodName, clazzRef, fromClass = false) }
+          val suggestionsFromFields = filterMapByName(to.fields, p.getName).toList.map { case (methodName, clazzRef) => ExpressionSuggestion(methodName, clazzRef, fromClass = false, None, Nil) }
           val suggestionsFromClass = typeDefinitions.get(to.objType.klass).map(c => filterClassMethods(c, p.getName, staticContext = false, fromClass = suggestionsFromFields.nonEmpty)).getOrElse(Nil)
           Future.successful(suggestionsFromFields ++ suggestionsFromClass)
         case TypingResultWithContext(tu: TypedUnion, staticContext) => Future.successful(tu.possibleTypes.map(_.objType.klass).flatMap(klass => typeDefinitions.get(klass).map(c => filterClassMethods(c, p.getName, staticContext)).getOrElse(Nil)))
         case TypingResultWithContext(td: TypedDict, _) => dictQueryService.queryEntriesByLabel(td.dictId, if (shouldInsertDummyVariable) "" else p.getName)
-          .map(_.map(list => list.map(e => ExpressionSuggestion(e.label, td, fromClass = false)))).getOrElse(successfulNil)
+          .map(_.map(list => list.map(e => ExpressionSuggestion(e.label, td, fromClass = false, None, Nil)))).getOrElse(successfulNil)
       }.getOrElse(successfulNil)
     }
 
     def filterClassMethods(classDefinition: ClazzDefinition, name: String, staticContext: Boolean, fromClass: Boolean = false): List[ExpressionSuggestion] = {
       val methods = filterMapByName(if (staticContext) classDefinition.staticMethods else classDefinition.methods, name)
 
       methods.values.flatten
-        .map(m => ExpressionSuggestion(m.name, m.signatures.head.result, fromClass = fromClass))
+        .map(m => ExpressionSuggestion(m.name, m.signatures.head.result, fromClass = fromClass, m.description, m.signatures.head.noVarArgs.map(p => Parameter(p.name, p.refClazz))))
```

Comment:
Shouldn't be `m.signatures.toList.maxBy(_.parametersToList.length)` as it is in `UIProcessObjectsFactory.toUIMethod`? Also let's add `TODO: present all overloaded methods on FE`

Reply:
1. Ok, changed that.
2. Sure, done that.
3. Not sure about that. The only common code is: `signatures.toList.maxBy(_.parametersToList.length)` and `signature.noVarArgs ::: signature.varArg.toList` and we intend to change that to show all methods.
4. Done

Reply:
ok